bulk/node_bulk_load-AssetRecord.csv

```diff
@@ -54,67 +93,30 @@ CGVEH-GLDROO-00477,A00964,Node,1,Open Ocean Glider 477,OPEN OCEAN GLIDER 1000M,T
-ATAPL-71403-00002,ATAPL-71403-00002,Node,0,Cabled Deep Profiler Mooring Wire-Following Vehicle,"MMP Extended Frame (no sensors), Glass Flotation (2 spheres), Ti controller housing extended",UW-APL,71403,SN00002,N/A,20140409,59190,,University of Washington
-ATAPL-71403-00003,ATAPL-71403-00003,Node,0,Cabled Deep Profiler Mooring Wire-Following Vehicle,"MMP Extended Frame (no sensors), Glass Flotation (2 spheres), Ti controller housing extended",UW-APL,71403,SN00003,N/A,20140409,59190,,University of Washington
-ATAPL-71403-00004,ATAPL-71403-00004,Node,0,Cabled Deep Profiler Mooring Wire-Following Vehicle,"MMP Extended Frame (no sensors), Glass Flotation (2 spheres), Ti controller housing extended",UW-APL,71403,SN00004,N/A,20140421,59190,,University of Washington
-ATAPL-71403-00005,ATAPL-71403-00005,Node,0,Cabled Deep Profiler Mooring Wire-Following Vehicle,"MMP Extended Frame (no sensors), Glass Flotation (2 spheres), Ti controller housing extended",UW-APL,71403,SN00005,N/A,20150428,59190,,University of Washington
-ATAPL-71403-00006,ATAPL-71403-00006,Node,0,Cabled Deep Profiler Mooring Wire-Following Vehicle,"MMP Extended Frame (no sensors), Glass Flotation (2 spheres), Ti controller housing extended",UW-APL,71403,SN00006,N/A,20150428,59190,,University of Washington
-ATAPL-71403-00007,ATAPL-71403-00007,Node,0,Cabled Deep Profiler Mooring Wire-Following Vehicle,"MMP Extended Frame (no sensors), Glass Flotation (2 spheres), Ti controller housing extended",UW-APL,71403,SN00007,N/A,20150428,59190,,University of Washington
-ATAPL-71533-00001,ATAPL-71533-00001,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00001,,,,,University of Washington
-ATAPL-71533-00002,ATAPL-71533-00002,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00002,,,,,University of Washington
-ATAPL-71533-00003,ATAPL-71533-00003,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00003,,,,,University of Washington
-ATAPL-71533-00004,ATAPL-71533-00004,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00004,,,,,University of Washington
-ATAPL-71533-00005,ATAPL-71533-00005,Node,0,Cabled Deep Profiler Mooring Docking Station,DP Docking Station,UW-APL,71533,SN00005,,,,,University of Washington
+CGVEH-GWFPGX-77401,A01080,Node,0,Global Wire-Following Profiler,GLOBAL WIRE FOLLOWING PROFILER,McLane,MMP-210-3512,ML12774-01,,20140925,98735.1,,Woods Hole Oceanographic Institution
+CGVEH-GWFPGX-93602,A00267,Node,0,Global Wire-Following Profiler,WFP GLOBAL,McLane,MMP-210-3512,ML12936-02,,20130320,111181.36,,Woods Hole Oceanographic Institution
+CGVEH-GWFPGX-99701,A00268,Node,0,Global Wire-Following Profiler,WFP GLOBAL,McLane,MMP-210-3512,ML12997-01,,20130320,97150.52,,Woods Hole Oceanographic Institution
```

Comment:
I don't see "CGVEH-GLDROO-00470" in the preview that Github shows me for file changes. But I confirmed it was there by clicking "view file" and it's on line 91. Looks good.

Reply:
Same here - weird!